persistence/azure-storage/configuration.md

```diff
@@ -26,8 +26,7 @@ The following settings are available for changing the behavior of saga persisten
   * NServiceBus.Azure defaults to `UseDevelopmentStorage=true`.
   * NServiceBus.Persistence.AzureStorage Version 1 defaults to `null`.
  * `CreateSchema`: Instructs the persister to create the table automatically. Defaults to `true`.
- * `AssumeSecondaryIndicesExist`: (Added in Version 1.4) By default every new saga scans for a secondary index. To avoid unnecessary performance tax, this option allows to skip the scans assuming all saga instances are in valid form and have secondary indices. 
-
+ * `AssumeSecondaryIndicesExist`: (Added in Version 1.4) Disables scanning for secondary index records when checking if a new saga should be created. A secondary index record was not created by the persister contained in the `NServiceBus.Azure` package. To provide backwards compatibilty, the `NServiceBus.Persistence.AzureStorage` package performs a full table scan, across all partitions, for secondary index records before creating a new saga. For systems that have only used the `NServiceBus.Persistence.AzureStorage` library, or have verified that all saga instances have a secondary index record, can safely disable the full table scan by using this configuration setting.
```

Comment:
We should add this info to the performance tuning document as well https://docs.particular.net/persistence/azure-storage/performance-tuning